### ISSUE TITLE
yarn start failed, please check

### ISSUE THREAD

**Issue by xiaods** [Conflux-Chain/conflux-developer-site#8]
```
❯ yarn start
yarn run v1.22.0
$ yarn genconf && docusaurus start
$ node development/gen_each_lang_sidebars_config.js
Error: Can't find markdown file with the id: introduction under conflux-portal
    at /Users/xiaods/Documents/Code/conflux-developer-site/development/gen_each_lang_sidebars_config.js:50:21
    at eachSidebarDoc (/Users/xiaods/Documents/Code/conflux-developer-site/development/gen_each_lang_sidebars_config.js:35:23)
    at eachSidebarDoc (/Users/xiaods/Documents/Code/conflux-developer-site/development/gen_each_lang_sidebars_config.js:33:29)
    at /Users/xiaods/Documents/Code/conflux-developer-site/development/gen_each_lang_sidebars_config.js:113:11
    at Array.map (<anonymous>)
    at /Users/xiaods/Documents/Code/conflux-developer-site/development/gen_each_lang_sidebars_config.js:108:29
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by yqrashawn** [comment]
Fixed in 8972a2dde907cb0e5e1c78a6f8b087b714b0e62d.

Add a new `REINIT` command.
The command will clear any uncommited changes and update everything to the same state as `origin/master`.

**Comment by xiaods** [comment]
thanks
